Source/Scene/ModelInstanceCollection.js

```diff
@@ -876,12 +884,42 @@ define([
         }
     }
 
-    ModelInstanceCollection.prototype.update = function(frameState) {
-        if (frameState.mode !== SceneMode.SCENE3D) {
-            oneTimeWarning('Instanced models in 2D', 'Instanced models are only supported in 3D.');
-            return;
-        }
+    var swizzleMatrix = new Matrix4(
+        0.0, 0.0, 1.0, 0.0,
+        1.0, 0.0, 0.0, 0.0,
+        0.0, 1.0, 0.0, 0.0,
+        0.0, 0.0, 0.0, 1.0
+    );
+
+    var scratchCartographic = new Cartographic();
+    var scratchCartesian = new Cartesian3();
+    var scratchCenter = new Cartesian3();
+    var scratchMatrix = new Matrix4();
+    var scratchRotation = new Matrix3();
+
+    function getRtcTransform2D(frameState, rtcTransform, result) {
```

Comment:
I wish I realized this ahead of time... anyways is the version here functionally equal to the `Transforms.basisTo2D`, could that be simplified with the code here?